### DISCUSSION TITLE
with clickhouse db, dbeaver can't get the table comment and show it as empty

### DISCUSSION THREAD

**Discussion by flyly0755** [dbeaver/dbeaver#19265] [Q&A]
```sql
CREATE TABLE diagtest2.commenttest
(

    `Name` String COMMENT 'comment for column'
)
ENGINE = MergeTree
PRIMARY KEY Name
ORDER BY Name
SETTINGS index_granularity = 8192
COMMENT 'comment for table'
```
With sql as above, dbeaver show table description of commenttest as empty on **Properties** tab.
For compare, on sub tab **Columns**, show Description of Name as _comment for column_, which 
meets the expectation
dbeaver version: 21.2.2.202110030714
clickhouse version:  21.12.3.32

**Comment by LonwoLonwo** [comment]
Hello @flyly0755 

Please update DBeaver version to the latest.

Table descriptions work fine for me in DBeaver 23.0.0:

![2023-03-06 14_11_27-DBeaver Ultimate 23 0 0 - actor](https://user-images.githubusercontent.com/45152336/223094764-506573d3-2fb8-4f1a-b9ad-978fd227b38f.png)

**Reply by flyly0755** [reply]
Thx, took a try and it's ok:)
